libraries/Microsoft.Bot.Builder.AI.QnA/Models/QnAMakerTraceInfo.cs

```diff
@@ -93,5 +93,21 @@ public class QnAMakerTraceInfo
         /// </value>
         [JsonProperty("qnaId")]
         public int QnAId { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether gets or sets environment of knowledgebase to be called. 
+        /// </summary>
+        /// <value>
+        /// A value indicating whether to call test or prod environment of knowledgebase. 
+        /// </value>
+        public bool IsTest { get; set; }
```

Comment:
This seems limiting - only 2 environments, ever. Could this instead be "endpoint" and just be a string? That way things like flighting, A/B Testing, and other features can be added later.

Reply:
As our current design, we have only two environment and it is well documented and well known by all our customer. Changing the name would confuse our users.